### DISCUSSION TITLE
Show some appreciations here

### DISCUSSION THREAD

**Discussion by alanxone** [noxone/regex-generator#196] [Show and tell]
Regex is always something bothering me, and even I've learned several simple patterns, I'm still in a mess when trying to use them in complex scenarios.

The generator saves me lots of time, and I'm here to say "Well done, and thank you!".

**Comment by noxone** [comment]
Hi Andrew, I'm very pleased to hear that. Thank you very much for your feedback.
I hope that this project can help many more people in the future and provide a first introduction to Regex.
